pkgs/tools/security/ghidra/extensions/wasm/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGhidraExtension,
+  ghidra,
+  ant,
+}:
+let
+  version = "2.3.1";
+in
+buildGhidraExtension {
+  pname = "wasm";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "nneonneo";
+    repo = "ghidra-wasm-plugin";
+    rev = "v${version}";
+    hash = "sha256-aoSMNzv+TgydiXM4CbvAyu/YsxmdZPvpkZkYEE3C+V4=";
+  };
+
+  # compile sleigh
```

Comment:
compiling should be done in buildPhase

Reply:
I don't want to override buildPhase from `buildGhidraExtension`. It does it's part.